PYME/recipes/processing.py

```diff
@@ -1959,3 +1959,103 @@ def execute(self, namespace):
             pass
             
         namespace[self.output_name] = res
+
+@register_module('DarkAndVarianceMap')
+class DarkAndVarianceMap(ModuleBase):
+    input = Input('input')
+    output_variance = Output('variance')
+    output_dark = Output('dark')
+    dark_threshold = Float(10000)  # this really should depend on the gain mode (12bit vs 16 bit etc)
+    variance_threshold = Float(90000)  # again this is currently picked fairly arbitrarily
+    start = Int(0)
+    end = Int(0)
+    uniform = Bool(False)  # simulate a uniform map
+
+    def execute(self, namespace):
+        from PYME.IO.MetaDataHandler import NestedClassMDHandler
+        from PYME.Analysis import gen_sCMOS_maps
+
+        logging.basicConfig() # without it got 'No handlers could be found for logger...'
+
+        default_sensor_size = (2048,2048) # we currently assume this is correct but could be chosen based
```

Comment:
If we are revisiting this, we should update this to use metadata entries (`"Camera.SensorWidth"`, `"Camera.SensorHeight"`)

Reply:
Oops, see that this is done below, ordering is a bit iffy.